### DISCUSSION TITLE
how does neuron compare to org-roam, org-brain and zetteldeft?

### DISCUSSION THREAD

**Discussion by AleXoundOS** [srid/neuron#610] [General]
As an inexperienced zettelkasten user I'm having trouble to choose one since the readme does not feature the comparison.

**Comment by srid** [comment]
Neuron is, from the get go, meant to satisfy one or both of the two specific use cases

- You use Markdown, and wish to control which text editor to use to edit them.
- You wish to publish your Markdown Zettelkasten notes as a static site.

There is no requirement that you use Emacs. You can even use [notepad.exe](https://en.wikipedia.org/wiki/Microsoft_Notepad) if that's what you prefer.

**Reply by AleXoundOS** [reply]
Alright, these points are pretty clear and mostly emphasize user interface aspects of the chosen variant of zettelkasten method. As for emacs I'm ok with being stick to it (prefer spacemacs).
It's also interesting what are the differences on the zettelkasten method side itself. I'm not yet quite familiar with terminology but I guess some of the interesting traits in no particular order: types of entries (org-brain has 2), directories-subdirectories handling, IDs handling, link types and relationships, special tags, search engine, timestamps, maintaining (whatever it means).
